plugins/services/src/js/components/modals/ServiceActionDisabledModal.js

```diff
@@ -223,27 +224,16 @@ class ServiceActionDisabledModal extends React.Component {
     const command = this.getUpdateCommand();
 
     return (
-      <div>
-        <p>
-          {intl.formatMessage({
-            id: "SERVICE_ACTIONS.SDK_SERVICE_UPDATE_PART_1"
-          })}
-          {" "}
-          <a
-            href={MetadataStore.buildDocsURI(
-              "/usage/managing-services/config-universe-service"
-            )}
-            target="_blank"
-          >
-            options.json
-          </a>
-          .{" "}
-          {intl.formatMessage({
-            id: "SERVICE_ACTIONS.SDK_SERVICE_UPDATE_PART_2"
-          })}
-        </p>
-        {this.getClipboardTrigger(command)}
-      </div>
+      <MountService.Mount
+        type={"ServiceEditMessage:Modal"}
+        getClipboardTrigger={this.getClipboardTrigger}
+        command={command}
+        intl={intl}
+      >
+        <div>
+          Only available on EE.
```

Comment:
Although we spoke offline - I'm making a note to get appropriate copy from @ashenden. My suggestion would be:
```
Editing this service is only available on <a href="https://mesosphere.com/product/">Mesosphere Enterprise DC/OS</a>.
```